app/src/test/java/com/distillery/interview/ui/HourlyWeatherViewModelTests.kt

```diff
@@ -55,8 +54,8 @@ class HourlyWeatherViewModelTests {
     @Test
     fun getHourlyWeather_returnErrorResponse() = runBlockingTest {
         //GIVEN
-        val mockError = Result.Error()
-        whenever(weatherRepository.getHourlyWeather()).thenReturn(mockError)
+        val mockError = RuntimeException()
+        whenever(weatherRepository.getHourlyWeather()).thenThrow(mockError)
```

Comment:
We can simulate an `IOException` since is more like to happen than a `RuntimeException`.